Source/ios/Storyboard/UIResponder+TyphoonOutletTransfer.m

```diff
@@ -0,0 +1,126 @@
+                    [self transferConstraintOutletForKey:propertyName
+                                                fromView:view];
+                }
+                // IBOutlet​Collection
+                if ([NSClassFromString(propertyType) isSubclassOfClass:[NSArray class]]) {
+                    [self transferConstraintOutletsForKey:propertyName
+                                                 fromView:view];
+                }
+            }
+        }
+    }
+    free(properties);
+}
+
+- (void)transferConstraintOutletForKey:(NSString *)propertyName
+                              fromView:(UIView *)view
+{
+    NSLayoutConstraint *constraint = [self valueForKey:propertyName];
+    if (constraint.typhoonTransferIdentifier) {
+        for (NSLayoutConstraint *transferConstraint in view.constraints) {
+            BOOL equalObjects = constraint == transferConstraint;
+            BOOL equalIdentifier = [constraint.typhoonTransferIdentifier isEqualToString:transferConstraint.typhoonTransferIdentifier];
+            if (!equalObjects && equalIdentifier) {
+                [self setValue:transferConstraint
+                        forKey:propertyName];
+            }
+        }
+    }
+}
+
+- (void)transferConstraintOutletsForKey:(NSString *)propertyName
+                               fromView:(UIView *)view
+{
+    NSPredicate *predicate = [NSPredicate predicateWithFormat:@"self isKindOfClass: %@",
+                              [NSLayoutConstraint class]];
+    NSArray *constraints = [self valueForKey:propertyName];
+    NSArray *filtered = [constraints filteredArrayUsingPredicate:predicate];
+    if (filtered.count > 0) {
+        BOOL needChange = NO;
+        NSMutableArray *newOutlets = [NSMutableArray new];
+        for (id outlet in constraints) {
+            id changeOutlet = outlet;
+            if ([outlet isMemberOfClass:[NSLayoutConstraint class]]) {
+                NSLayoutConstraint *constraint = outlet;
+                if (constraint.typhoonTransferIdentifier) {
+                    for (NSLayoutConstraint *transferConstraint in view.constraints) {
+                        BOOL equalObjects = constraint == transferConstraint;
+                        BOOL equalIdentifier = [constraint.typhoonTransferIdentifier isEqualToString:transferConstraint.typhoonTransferIdentifier];
+                        if (!equalObjects && equalIdentifier) {
+                            changeOutlet = transferConstraint;
+                            needChange = YES;
+                        }
+                    }
+                }
+            }
+            [newOutlets addObject:changeOutlet];
+        }
+        
+        if (needChange) {
+            [self setValue:newOutlets
+                    forKey:propertyName];
+        }
+        
+    }
+}
+
+static const char *getPropertyType(const char * attributes)
+{
+    char buffer[1 + strlen(attributes)];
```

Comment:
allocating unknown buffer size on stack is dangerous and it's potential security issue. All these helpers already exists in Typhoon and I'm not sure about this duplication at all. See TyphoonIntrospectionUtils